Mathlib/Algebra/Order/Field/Basic.lean

```diff
@@ -243,6 +243,14 @@ theorem div_le_one_of_le (h : a ≤ b) (hb : 0 ≤ b) : a / b ≤ 1 :=
   div_le_of_nonneg_of_le_mul hb zero_le_one <| by rwa [one_mul]
 #align div_le_one_of_le div_le_one_of_le
 
+/-- `a * b⁻¹` version of `div_le_one_of_le` -/
+lemma mul_inv_le_one_of_le (h : a ≤ b) (hb : 0 ≤ b) : a * b⁻¹ ≤ 1 := by
+  simpa only [← div_eq_mul_inv] using div_le_one_of_le h hb
+
+/-- `b⁻¹ * a` version of `div_le_one_of_le` -/
```

Comment:
Those docstrings read like "Here's the reason I made a PR", which is not very useful on declarations in Lean code. They would fit better in your PR description
```suggestion
lemma mul_inv_le_one_of_le (h : a ≤ b) (hb : 0 ≤ b) : a * b⁻¹ ≤ 1 := by
  simpa only [← div_eq_mul_inv] using div_le_one_of_le h hb

```